penguin/local_auth.py

```
@@ -0,0 +1,100 @@
+"""Shared local auth helpers for Penguin browser and TUI bootstrap flows."""
+
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import Mapping
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)